README.md

```diff
@@ -1,19 +1,86 @@
+    <img src="https://github.com/apollographql/apollo-ios-dev/actions/workflows/ci-tests.yml/badge.svg?branch=main" alt="GitHub Action Status">
+  </a>
+  <a href="https://raw.githubusercontent.com/apollographql/apollo-ios/main/LICENSE">
+    <img src="https://img.shields.io/badge/license-MIT-lightgrey.svg?maxAge=2592000" alt="MIT license">
+  </a>
+  <a href="https://github.com/apple/swift">
+    <img src="https://img.shields.io/badge/Swift-5.7-orange.svg" alt="Swift 5.7 supported">
+  </a>
+</p>
 
-This project uses [Tuist](https://tuist.io/) to handle generation of the Xcode project/workspace for development. In order to use Tuist run the following command in terminal to ensure you have it installed:
+### Apollo iOS Dev
+
+This repo contains the development environment for working on and contributing to the Apollo iOS ecosystem. This repo uses a git subtree setup to include the following repos for development:
+
+- [apollo-ios](https://github.com/apollographql/apollo-ios)
+- [apollo-ios-codegen](https://github.com/apollographql/apollo-ios-codegen)
+
+All code changes are pushed out to their respective repos whenever a PR is merged. This allows us to provide a cohesive development environment while also providing packages to users that contain less overall files and dependencies, such as things only really used for development and testing.
+
+## Getting Started
+
+To get started contributing to Apollo iOS, the first step you need to take is to fork this (apollo-ios-dev) repo. Once you have forked and cloned the repo the next step is to generate the Xcode Workspace that you will use for development.
+
+### Tuist
+
+This project uses [Tuist](https://tuist.io/) to handle generation of the Xcode workspace for development. In order to use Tuist run the following command in terminal to ensure you have it installed:
 
 ```
 curl -Ls https://install.tuist.io | bash
 ```
 
 In order to generate the project/workspace you will need to run the [tuist generate](https://docs.tuist.io/commands/generate) command from the project root.
 
-There is also a githook setup to auto-run `tuist generate` whenever a new branch is checked out, in order for git to find and use the hook run the following make command to ensure the git config is pointing to the correct location:
+There is also a githook setup to auto-run `tuist generate` whenever a branch is checked out, in order for git to find and use the hook run the following make command to ensure the git config is pointing to the correct location:
 
 ```
 make repo-setup
 ```
 
-A [Get started](https://docs.tuist.io/tutorial/get-started) guide for Tuist along with [other documentation](https://tuist.github.io/tuist/main/documentation/projectdescription/project) is also available for reference.
\ No newline at end of file
+> Note: This will run the following command updating your git config `git config core.hooksPath .githooks`
```

Comment:
Let's make this more clear that we aren't altering your global git config

```suggestion
> Note: This will updating the local git config in your checkout of the repository by run the following command : `git config core.hooksPath .githooks`
```